### PR TITLE
adding collection script for validation and mutatin admission webhooks

### PR DESCRIPTION
**Pull request by sferich888**
@sttts ptal

### CODE REVIEW

File: collection-scripts/gather_admission_webhooks [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="/must-gather"
+WEB_HOOK_PATH="${BASE_COLLECTION_PATH}/web_hooks/"
+
+mkdir ${WEB_HOOK_PATH}
```
Comment (sttts) [inline collection-scripts/gather_admission_webhooks]: maybe `mkdir -p` ? Or are we sure that dir does not exist?

Comment (sferich888) [reply collection-scripts/gather_admission_webhooks]: fixed.

**Comment by sferich888** [comment]
/remove-lifecycle stale
I had to rebase due to an extensive amount of changes in the scripts. @sttts please take another look.

**Comment by sttts** [comment]
Needs a BZ.

**Pull request merged**